zimbra-x-webclient.adoc

```diff
@@ -0,0 +1,93 @@
+= Installing {product-short} X Webclient
+
+Optionally you can install {product-short} X Webclient, which is currently in beta stage and you can provide early feedback to {product-short}.
+Below are the steps to setup {product-short} X Webclient on {product-short}
+
+. Run {product-short} X Webclient on any of the mailbox server
+
+	* Make sure latest version of **NodeJS** and **NPM** is installed
+	* In case of multinode environment make sure you are selecting a node which is currently working as a mailbox node
+	* Clone respository of {product-short} X Webclient by executing `git clone https://github.com/Zimbra/zm-x-web.git`
+	* If you want to do any customizations on top of {product-short} X Webclient then you can do as shown here https://github.com/Zimbra/zm-x-web/wiki/Customising-Zimbra-X-Webclient[Customising-Zimbra-X-Webclient]
+	* Install npm modules by executing `npm install`
+	* Create a production build by executing `npm run build`
```

Comment:
If they customized a folder other than `clients/default` they will need to specify `CLIENT=foo` or whatever their client folder is when they build

Reply:
Fixed